src/components/button-toggle/button-toggle.stories.mdx

```diff
@@ -39,14 +39,14 @@ import ButtonToggleGroup from "carbon-react/lib/components/button-toggle-group";
 
```

Comment:
Maybe worth moving this to a `tsx` file so we can get the benefit of type checking, it's probably out of scope for this ticket so don't worry about it you'd prefer to leave it for now.

Reply:
see reply above